internal/workflow_testsuite.go

```diff
@@ -999,10 +1167,10 @@ func (e *TestWorkflowEnvironment) AssertExpectations(t mock.TestingT) bool {
 // It can produce a false result when an argument is a pointer type and the underlying value changed after calling the mocked method.
 func (e *TestWorkflowEnvironment) AssertCalled(t mock.TestingT, methodName string, arguments ...interface{}) bool {
 	dummyT := &testing.T{}
-	if !(e.workflowMock.AssertCalled(dummyT, methodName, arguments...) || e.activityMock.AssertCalled(dummyT, methodName, arguments...)) {
-		return e.workflowMock.AssertCalled(t, methodName, arguments...) && e.activityMock.AssertCalled(t, methodName, arguments...)
-	}
-	return true
+	return e.AssertWorkflowCalled(dummyT, methodName, arguments...) ||
+		e.AssertActivityCalled(dummyT, methodName, arguments...) ||
```

Comment:
Why are some of these called with `dummyT` and some `t`?

Reply:
That's a great question, and I have no idea. I just copied the existing calls.

Reply:
I think we can just leave it as is, I will note I plan to deprecate these functions in the next minor release because they cause a lot of confusion for users since they awkwards combine workflow and activities so I wouldn't spend a lot of time fussing over them.